defaults/main.yaml

```diff
@@ -15,4 +15,18 @@ preserve_job_vars:
 # Default provision check interval
 provision_check_interval: 5m
 
-babylon_job_template: "{{ vars.anarchy_governor.vars.job_vars.__meta__.tower.job_template | default('job-runner') }}"
+# AnarchyGovernor controls __meta__ settings.
+__meta__: "{{ vars.anarchy_governor.vars.job_vars.__meta__ }}"
+
+tower_organization_name: >-
+  {{ __meta__.tower.organization | default('babylon') }}
```

Comment:
don't you want `vars.__meta__` ? otherwise i foresee "undefined variable" errors maybe?

Reply:
It is already covered by:

```
__meta__: "{{ vars.anarchy_governor.vars.job_vars.__meta__ }}"
```

Example:

```
- hosts: localhost
  gather_facts: false
  vars:
    anarchy_governor:
      vars:
        job_vars:
          __meta__:
            tower:
              organization: foo
        undefined_var: "{{ this_is_not_defined }}"
    __meta__: "{{ vars.anarchy_governor.vars.job_vars.__meta__ }}"
    tower_organization_name: >-
      {{ __meta__.tower.organization | default('babylon') }}
  tasks:
  - debug:
      msg: "{{ tower_organization_name }}"
```